resources/services/cloudbilling/services.go

```diff
@@ -13,7 +13,7 @@ func Services() *schema.Table {
 	return &schema.Table{
 		Name:          "gcp_cloudbilling_services",
 		Description:   "Encapsulates a single service in Google Cloud Platform",
-		Resolver:      fetchCloudbillingServices,
+		Resolver:      client.RequireEnabledServices(fetchCloudbillingServices, "cloudbilling.googleapis.com"),
```

Comment:
This doesn't actually work here. It works for accounts table which is multiplexed over projects, but here it is kinda global, so `RequireEnabledServices` doesn't know where to look (client.ProjectID is empty, and disabled apis/services seems like a project level configuration).
So, cloudbilling accounts are not fetched if Cloud Billing API is disabled, but services still gives the error.

How to handle this? Just catch and ignore the error?

@bbernays @disq